app/forms/edit_item_form.py

```diff
@@ -0,0 +1,36 @@
+import requests
```

Comment:
This seems to be an external package but it is not in our dependencies. Make sure to install this so it gets added to the `Pipfile` and then export the new `requirements.txt`